### PR TITLE
[tapocontrol] fix "device not found" with L510/L530 hw rev2 / add L920 and P110 things

### PR DESCRIPTION
**Pull request by wildcs**
This PR fixes the "wrong device found" error for Tapo L510 and L530 Bulbs with Hardware Rev.2
Cause: Tapo renamed types from "L530_Series" to "L530"

Also implemented L920 LED-Lightstrip and P110 SmartPlug with basic functions.

### CODE REVIEW

**Comment by lolodomo** [comment]
This is a breaking change as some thing types are renamed. This will be a problem for users who defined such thing types.

**Comment by wildcs** [comment]
> This is a breaking change as some thing types are renamed. This will be a problem for users who defined such thing types.
Yes youre right. But in my oppinion this change is more a "bugfix" than "enhancement". 
It's a new plugin released with OH3.2. so this issue with L510/L530-Things may be a result of the young release time.
It's the better way to handle the thing names unique than add multiple thing-types for the same device.

Of course the new thing types for P110 and L920 is an enhancement.

**Comment by lolodomo** [comment]
Yes, I am just saying that it has to be mentioned in the 3.3 release notes with an explanation of what changed and what to do.

File: bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/L510.xml
Comment (lolodomo) [inline bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/L510.xml:7]: Do you want to remove the E?

File: bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/L920.xml
Comment (lolodomo) [inline bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/L920.xml:7]: Please fix the comment.

File: bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml
Comment (lolodomo) [inline bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml:3]: Please update the default translation properties in i18n folder to be in sync with all your changes

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by wildcs** [comment]
> Yes, I am just saying that it has to be mentioned in the 3.3 release notes with an explanation of what changed and what to do (by the user).

Mhh. If it not will updated before 3.3 release maybe it'll be though better to let the old thing-types for this devices and do a code workaround for devices without "series"..... 
i'm not sure what's better now

File: bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml
Comment (wildcs) [reply bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml:3]: updated default translation properties in i18n
`thing-type.tapocontrol.L510.label = L510 Series White-Bulb
thing-type.tapocontrol.L510.description = Tapo Smart dimmable White-Light-Bulb
thing-type.tapocontrol.L530.label = L530 Series Color-Bulb
thing-type.tapocontrol.L530.description = Tapo Smart Multicolor Light-Bulb`

Comment (lolodomo) [reply bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml:3]: You did not add entries for your new thing types.

Ideally, you should just run the command: `mvn i18n:generate-default-translations`

Comment (wildcs) [reply bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml:3]: Did `mvn clean install`. I never had to do this or edit this file manually before....

When running ` mvn i18n:generate-default-translations` i got an warning and error: `[WARNING] The POM for org.openhab.core.tools:i18n-maven-plugin:jar:3.3.0-SNAPSHOT is missing`
If i change the openhab-addons\pom.xml from `${ohc.version}` to `3.2.0` generating translations works fine.
```<plugin>
          <groupId>org.openhab.core.tools</groupId>
          <artifactId>i18n-maven-plugin</artifactId>
          <version>${ohc.version}</version>    <----- change this to 3.2.0
    </plugin>
```
changed it back and did clean install again. now my translation properties are fixed

**Comment by lolodomo** [comment]
> Mhh. If it not will updated before 3.3 release maybe it'll be though better to let the old thing-types for this devices and do a code workaround for devices without "series".....
> i'm not sure what's better now

I let you decide but if you break the compatibility of things with your enhancement, I prefer you mention it in the appropriate file to inform the users when they will move to openHAB 3.3 (and to integrate it in the 3.3 release notes).
It could also depend on how many users you thing will use this binding and these changed thing types. This is probably not the same decision if you think only few of them will be impacted or if it could impact several thousands of users.

**Review by lolodomo — APPROVED** [review]
Code LGTM

**Comment by lolodomo** [comment]
Waiting for your decision before merging.

**Comment by wildcs** [comment]
> I prefer you mention it in the appropriate file to inform the users when they will move to openHAB 3.3 (and to integrate it in the 3.3 release notes).

Yes merge it. It's better to change it in this early version. 
Dont' know how many people will use this binding. But if they buy new devices they will probably get new HW-rev anyway - which will not work with th 3.2 release.

> I prefer you mention it in the appropriate file to inform the users when they will move to openHAB 3.3 (and to integrate it in the 3.3 release notes).

Can you tell me where i should mention it?

**Comment by lolodomo** [comment]
Please add a section for v3.3 and one entry for your binding in that section in this file:
https://github.com/openhab/openhab-distro/blob/main/distributions/openhab/src/main/resources/bin/update.lst

**Pull request merged**
